### ISSUE TITLE
shellscript TextMate grammar incorrectly tokenizes here doc

### ISSUE THREAD

**Issue by alexr00** [atom/language-shellscript#139]
```
cat <<"some delim" > some_file
some
file
content
some delim
false 
```

The highlighting fails on the space between the 1st `some` and `delim`.
`delim` is not seen as part of the _word_. 
`man bash` says:

>  If any characters in word are quoted, the delimiter is the result of quote removal on word,[...]

Workaround: Remove the space in the `word`.

Originally opened [here](https://github.com/Microsoft/vscode/issues/72512)
@GloobyBoolga

**Comment by 50Wliu** [comment]
Thanks; this also seems to be an issue with the tree-sitter grammar.

**Comment by jasonrudolph** [comment]
> this also seems to be an issue with the tree-sitter grammar.

Thanks for noting this, @50Wliu. :bow:

For anyone using the Tree-sitter grammar, #143 resolves this issue. That fix is included in language-shellscript v0.27.11, and the next release of Atom will include this fix.

### Before


<img width="523" alt="Screen Shot 2019-05-02 at 12 04 13 PM" src="https://user-images.githubusercontent.com/2988/57089614-d1c82e80-6cd2-11e9-8481-7926a05ef53b.png">

### After

<img width="523" alt="Screen Shot 2019-05-02 at 12 03 58 PM" src="https://user-images.githubusercontent.com/2988/57089613-d12f9800-6cd2-11e9-8fbf-9e9b85220b5c.png">

**Comment by 50Wliu** [comment]
Since this issue now applies only to the first-mate grammar, I'm closing this out.
